creategameprojects.sh

```diff
@@ -0,0 +1 @@
+devtools/bin/vpc_linux /2015 /define:WORKSHOP_IMPORT_DISABLE /define:SIXENSE_DISABLE /define:NO_X360_XDK /define:RAD_TELEMETRY_DISABLED /define:DISABLE_ETW /define:LTCG /no_ceg /retail /define:CERT /tf +game /mksln games.sln
```

Comment:
Change `/mksln games.sln` to `/mksln games.mak` for a properly named makefile.

Reply:
open pr on a pr xd

Reply:
or just `/mksln Makefile`

Reply:
oh it automatically adds `.mak` :/